content/docs/intro/concepts/state.md

```diff
@@ -13,124 +13,81 @@ menu:
 
-However, if you would prefer to manage this yourself, you can do so by opting
-into the filesystem or cloud storage backend.
-
-### Self-managed backends
-
-To use the [filesystem or local](#filesystem-or-local) option, run `pulumi login --local`
-to have Pulumi store your _checkpoint_ files locally on your machine. The default
-directory for these JSON files is `~/.pulumi`.
-
-To use the [cloud storage or remote](#cloud-storage-or-remote) option, run `pulumi login
---cloud-url <your-cloud-storage-url>` to store your checkpoint files using a
-supported cloud provider like AWS, Azure, or Google Cloud Platform. Checkpoint files are
-also stored in `.pulumi` directory. For example, if you were using the Amazon S3
-self-managed backend, your checkpoint files would be stored at
-`s3://<my-pulumi-state-bucket>/.pulumi` where `<my-pulumi-state-bucket>` represents the
-name of your S3 bucket.
-
-The following links offer additional details for setting up provider-specific
-command-line tools and storage accounts.
-
-{{< chooser cloud "aws,azure,gcp" / >}}
-
-{{< choosable cloud aws >}}
-<p></p>
-
-For **Amazon S3**, see [AWS Setup]({{< relref "/docs/intro/cloud-providers/aws/setup" >}})
-and [Working with Amazon S3 buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html). Also see [AWS Session](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/) for configuring credentials for this backend.
-
-{{< /choosable >}}
-
-{{< choosable cloud azure >}}
-<p></p>
-
-For **Azure Blob Storage**, see [Azure Setup]({{< relref "/docs/intro/cloud-providers/azure/setup" >}})
-and Microsoft's [Storage Blobs Quickstart](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli). This backend requires setting the environment variables `AZURE_STORAGE_ACCOUNT` and either `AZURE_STORAGE_KEY` or `AZURE_STORAGE_SAS_TOKEN`.
-
-{{< /choosable >}}
+- Robust state management, with transactional checkpointing for fault tolerance and recovery
+- Concurrent state locking to prevent corrupting your infrastructure state in a team environment
+- Full deployment history for auditing and rollback purposes
+- Encrypted state in transit and at rest
+- Managed encryption and key management for secrets
+- Secure access to cloud resource metadata, with client-side authentication to your cloud provider
+- Team policies, including Policy as Code and Role Based Access Control (RBAC)
 
-{{< choosable cloud gcp >}}
-<p></p>
+The Pulumi Service backend requires no additional configuration after [installing the CLI]({{< relref "/docs/get-started/install" >}}). Pulumi offers this backend hosted online as a free service for individuals with [advanced tiers]({{< relref "/pricing" >}}) for teams and enterprises (with <a href="https://app.pulumi.com/site/trial" target="_blank">free trials</a>. It has successfully undergone multiple security audits including SOC2, pen-testing, and more.
```

Comment:
We should link to the SOC2 announcement

Reply:
Definitely, AFAIK it's not set to go out until next week, however, and I'm hoping to have merged by then :smile: